examples/src/main/res/values/strings.xml

```diff
@@ -79,8 +79,11 @@
     <string name="title_navigation_view_fragment" translatable="false">NavigationView In A Fragment</string>
     <string name="description_navigation_view_fragment" translatable="false">Shows how to use NavigationView to implement turn by turn in a fragment.</string>
 
-    <string name="title_navigation_route_ui">Navigation Map Route</string>
-    <string name="description_navigation_route_ui">An example using NavigationMapRoute</string>
+    <string name="title_navigation_route_ui" translatable="false">Navigation Map Route</string>
+    <string name="description_navigation_route_ui" translatable="false">An example using NavigationMapRoute</string>
+
+    <string name="title_traffic_toggle" translatable="false">Traffic toggle</string>
+    <string name="description_traffic_toggle" translatable="false">Toggle the visibility of the NavigationView\'s traffic congestion layers.</string>
```

Comment:
This shouldn't be `translatable = "false"`. Actually none of the resources here shouldn't 👀 https://github.com/mapbox/mapbox-navigation-android/pull/3438#issue-468106657 https://github.com/mapbox/mapbox-navigation-android/issues/2909

Reply:
I've rebased this pr, so it's up-to-date with our main branch.

I added `translatable="false"` so that Android Studio doesn't throw the red underline warning on any string that doesn't have `translatable="false"`.

<img width="838" alt="Screen Shot 2020-08-17 at 3 28 33 PM" src="https://user-images.githubusercontent.com/4394910/90450397-7b852780-e09e-11ea-9f25-3782f682343f.png">




> Actually none of the resources here shouldn't 👀 #3438 (comment) #2909

What in https://github.com/mapbox/mapbox-navigation-android/pull/3438/files means that `translatable = "false"` shouldn't be in `examples/src/main/res/values/strings.xml` ?

Reply:
Let's remove `translatable = "false"` so that we have the flexibility for future internationalization of the test app and other strings.